PIM/UserInterface/04_DataTypeList.md

```diff
@@ -0,0 +1,615 @@
+
+- ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Read only*   
+  Enable this toggle to set the string read-only. Disable the toggle to set the string editable. By default, this toggle is disabled.
+
+
+## Currency
+
+![Currency](/Assets/Screenshots/PIM/DataTypes/Currency.png "[Currency]")
+
+The data type *Currency* is used to indicate a currency. A drop-down list with all currencies in the system is displayed in the product view.
+
+### Configuration
+
+This data type has no further configuration settings.
+
+
+## Language
+
+![Language](/Assets/Screenshots/PIM/DataTypes/Language.png "[Language]")
+
+The data type *Language* is used to indicate a language. A drop-down list with all languages in the system is displayed in the product view.
+
+### Configuration
+
+This data type has no further configuration settings.
+
+
+## Country
+
+![Country](/Assets/Screenshots/PIM/DataTypes/Country.png "[Country]")
+
+The data type *Country* is used to indicate a country. A drop-down list with all countries in the system is displayed in the product view. In the *PIM Basic Set*, the following attributes have the data type *Country*:
+- Country of origin
+
+### Configuration
+
+This data type has no further configuration settings.
+
+
+## TreeNode
+
+The data type *TreeNode* is used for indications with multiple, predefined selection options. You have to define the different selection options in the configuration. Additionally you can configure whether the selection of values will be a multiple or single selection.
+
+[comment]: <> (to be edited)
+
+![TreeNode](/Assets/Screenshots/PIM/DataTypes/TreeNode.png "[TreeNode]")
+
+### Configuration
+
+![TreeNode](/Assets/Screenshots/PIM/DataTypes/Configuration/TreeNode.png "[TreeNode]")
+
+- ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Multi Select*   
+  Enable this toggle to enable a multiple selection of values. Disable the toggle to allow only a single selection of values. By default, this toggle is disabled.
+
+> [Info] In the product view, the data type *TreeNode* is displayed either by a drop-down list or a a box with all created options. The drop-down list is only displayed for one dimensional single select trees.
+
+**Edit tree**
+
+- ![Add](/Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click first this button and then the plus sign displayed in the left column of the *Edit tree* table to add an attribute value. The *Add Element* window is displayed.
+
+### Add Element
+
+![Add element](/Assets/Screenshots/PIM/DataTypes/Configuration/AddElement.png "[Add element]")
+
+- *Name*   
+  Enter the attribute value name.
+
+- *Key*   
+  Enter the attribute value key.
+
+- [CANCEL]   
+  Click this button to cancel adding a value.
+
+- [SAVE]   
+  Click this button to save the value and add it to the table.
+
+
+## Completeness (internal, not editable)
+
+This data type is for internal use only. It is strongly advised not to use this data type without the required knowledge.   
```

Comment:
Shows the completeness of a product, according to the required fields in the AttributeSet.